webapp/channels/src/components/threading/virtualized_thread_viewer/thread_viewer_row.tsx

```diff
@@ -79,7 +80,13 @@ function ThreadViewerRow({
                 />
                 {replyCount > 0 && (
                     <div className='root-post__divider'>
-                        <div>{`${replyCount} Replies`}</div>
+                        <div>
```

Comment:
is this additional `div` related to styling constrains?

Reply:
Yes, it is to style the box in which the reply count and text will sit.

Reply:
![CleanShot 2024-03-14 at 07 01 02@2x](https://github.com/mattermost/mattermost/assets/11034289/b8772959-252f-4e57-b04a-f57e188f1c72)